controllers/openshift_controller.go

```diff
@@ -782,6 +782,7 @@ func (r *KataConfigOpenShiftReconciler) createRuntimeClass(runtimeClassName stri
 			},
 			ObjectMeta: metav1.ObjectMeta{
 				Name: runtimeClassName,
```

Comment:
```suggestion
				Name:       runtimeClassName,
```

to silent `make fmt` .